tools/release/bump-version-and-create-release-note.sh

```diff
@@ -23,7 +23,7 @@ VERSION=`cat VERSION`
 OUTPUT="docs/releases/${RELEASE_VERSION}.md"
 
 # generate release note
-docker run -it -v "`pwd`":/usr/local/src/your-app ferrarimarco/github-changelog-generator \
+docker run -it -v --rm "`pwd`":/usr/local/src/your-app ferrarimarco/github-changelog-generator \
```

Comment:
Why?

Reply:
It wasn't removing the container after being run, which was leaving me with a bunch of containers I would have to manually clean up.